index.dd

```diff
@@ -43,8 +43,13 @@ void main()
     import std.range, std.stdio;
 
     auto sum = 0.0;
-    auto count = stdin.byLine
-        .tee!(l => sum += l.length).walkLength;
+    auto count = stdin
+        //Get an input range set up to read one line at a time
+        .byLine
+        //Perform a transparent operation 
+        .tee!(l => sum += l.length)
+        //Optimized version of length
```

Comment:
this comment is misleading. In this case `.length` is not available since it isn't known in advance.

Reply:
How would you describe walkLength instead?

Reply:
It iterates the range and counts for each iteration. I'm not sure how to best word that.

Reply:
What about 
"Best-effort implementation of length" or "Best-effort implementation of length for ranges"?

Or maybe remove the comment? 🤔

Reply:
I think it would be fine with no comment, `walkLength` is a lot easier to figure out from the context than `tee`.